app/views/contexts/calculated_summary_context.py

```diff
@@ -45,7 +48,7 @@ def build_groups_for_section(
         ]
 
     def build_view_context_for_calculated_summary(
-        self, current_location: Location
+        self, current_location: Location, routing_path: RoutingPath
```

Comment:
You don't need to pass the path as you can assess it from the instance already?.

Reply:
The current location doesn't need to be passed either, it is already accessible.

Reply:
Updated this 👍